internal/api/opentelemetry-tracer_test.go

```diff
@@ -78,22 +78,16 @@ func (ts *OpenTelemetryTracerTestSuite) TestOpenTelemetryTracer_Spans() {
 		method1 := getAttribute(attributes1, semconv.HTTPMethodKey)
 		assert.Equal(ts.T(), "POST", method1.AsString())
 		url1 := getAttribute(attributes1, semconv.HTTPTargetKey)
-		assert.Equal(ts.T(), "http://localhost/something1", url1.AsString())
+		assert.Equal(ts.T(), "/something1", url1.AsString())
 		statusCode1 := getAttribute(attributes1, semconv.HTTPStatusCodeKey)
 		assert.Equal(ts.T(), int64(404), statusCode1.AsInt64())
 
-		userAgent1 := getAttribute(attributes1, semconv.HTTPUserAgentKey)
-		assert.Equal(ts.T(), "stripped", userAgent1.AsString())
```

Comment:
it seems like these are no longer included in the trace in the latest version. IIRC we initially patched these as the otel version included a stray user-agent header which stayed in memory.